checkstyle/suppressions.xml

```diff
@@ -371,4 +371,8 @@
 
      <!-- Add the new suppression rule for JaasTestUtils.java -->
     <suppress checks="ImportControl" files="(JaasTestUtils).java" />
+
+    <suppress checks="FinalLocalVariable" files="."/>
+    <suppress checks="FinalParameters" files="."/>
```

Comment:
From my POV both of these checks are overzealous and hence I turned them off, but I'm open to a discussion.

Related links:
- https://checkstyle.sourceforge.io/checks/misc/finalparameters.html
- https://checkstyle.sourceforge.io/checks/coding/finallocalvariable.html
- https://softwareengineering.stackexchange.com/questions/48413/in-java-should-i-use-final-for-parameters-and-locals-even-when-i-dont-have-t
- https://stackoverflow.com/questions/3635863/why-method-parameters-needs-to-be-set-as-final